server/routes/prisons/prisons.ts

```diff
@@ -62,37 +62,39 @@ export default function routes({ prisonService }: Services): Router {
     return res.redirect(`/prisons`)
   })
 
-  get('/:prisonId([A-Z]{3})/edit', async (req, res) => {
+  get('/:prisonId([A-Z]{3})/session-templates', async (req, res) => {
     const { prisonId } = req.params
 
     const { prison, prisonName } = await prisonService.getPrison(res.locals.user.username, prisonId)
     const message = req.flash('message')
 
-    res.render('pages/prisons/edit', { errors: req.flash('errors'), prison, prisonName, message })
+    res.render('pages/prisons/viewSessionTemplates', { errors: req.flash('errors'), prison, prisonName, message })
   })
 
-  post('/:prisonId([A-Z]{3})/edit', async (req, res) => {
+  post('/:prisonId([A-Z]{3})/activate', async (req, res) => {
```

Comment:
I've split the `/edit` POST route up to hopefully make it simpler. More generally, we can split this file up soon so not all routes are in one place....